code/android-core-library/src/phone/java/com/adobe/marketing/mobile/MobileCore.java

```diff
@@ -580,17 +568,13 @@ public static void updateConfiguration(final Map<String, Object> configMap) {
      * or {@link #configureWithFileInPath(String)} or {@link #configureWithFileInAssets(String)}
      */
     public static void clearUpdatedConfiguration() {
-        if (eventHub == null) {
-            Log.debug(LOG_TAG, "Failed to clear updated configuration (%s)", NULL_CONTEXT_MESSAGE);
-            return;
-        }
-
-        EventData eventData = new EventData();
-        eventData.putBoolean(CoreConstants.EventDataKeys.Configuration.CONFIGURATION_REQUEST_CONTENT_CLEAR_UPDATED_CONFIG,
+        Map<String, Object> eventData = new HashMap<>();
+        eventData.put(CoreConstants.EventDataKeys.Configuration.CONFIGURATION_REQUEST_CONTENT_CLEAR_UPDATED_CONFIG,
                 true);
-        final Event event = new Event.Builder("Clear updated configuration", EventType.CONFIGURATION,
-                EventSource.REQUEST_CONTENT).setData(eventData).build();
-        eventHub.dispatch(event);
+
+        Event event = new Event.Builder("Configuration Update", EventType.CONFIGURATION,
```

Comment:
Event name is changed, need to update `MobileCoreTests` if we want to use the new event name

Reply:
Nvm, just saw you have changed the tests